client/src/components/FilesDialog/FilesDialog.vue

```diff
@@ -457,5 +460,12 @@ onMounted(() => {
                 >
             </BAlert>
         </template>
+        <template v-slot:buttons>
```

Comment:
I feel like I would make it dependent on this feature being available.

Reply:
Oh shoot, yes, I was going to do that, somehow forgot to put it in draft and do that first, sorry about that.